### ISSUE TITLE
Custom load order no longer applied

### ISSUE THREAD

**Issue by Mithril-Leaf** [ThibautSF/ParadoxosModManager#58]
When using the custom load order feature currently, mods are made into local mods as appropriate. However once that is done, those mods are no longer sorted properly in the launcher, basically being randomly sorted. This feature previously worked, but something seems to have broken it.

**Comment by Shadowedcross** [comment]
I'm having the same issue, it's frustrating because I need the function for a collection I'm trying to use.

**Comment by Perhelion** [comment]
I used the tool first time (I've over 260 mods installed) and now I must see it is not working. Are the old versions not working to?

**Comment by Perhelion** [comment]
I see now the tool is fully missing the 2.5.* support for Stellaris. As the load order is stored in the ´`game_data.json`´ not in the ´`dlc_load.json´`

https://github.com/ThibautSF/ParadoxosModManager/blob/master/ParadoxGameModManager/supported_games.json

**Comment by Perhelion** [comment]
As I've yesterday learned Python I've written a script to fix this: 
```python
import json, os.path
try: to_unicode = unicode
except NameError: to_unicode = str
from pathlib import Path

#Must be adjusted
data_folder = Path(os.path.expanduser('~') + "\\Documents\\Paradox Interactive\\Stellaris")
load_order = str(data_folder / "dlc_load.json")
oList = []

#Read mods_registry JSON file
with open(str(data_folder / "mods_registry.json")) as data_file:
	data_loaded = json.load(data_file)
#Read dlc_load JSON file
with open(load_order) as data_file:
	load_order = json.load(data_file)

def reorder_managerId(regId, i):
	for id in data_loaded:
		d = data_loaded[id]
		if d["gameRegistryId"] == regId:
			oList.append(id)

load_order = load_order["enabled_mods"]
for i in range(0,len(load_order)):
	reorder_managerId(load_order[i],i)
			
#Write JSON file
with open(str(data_folder / "game_data.json"), 'w', encoding='utf-8') as f:
	json.dump({"modsOrder": oList}, f, ensure_ascii=False, indent=2)
```

**Comment by Vargtazz** [comment]
Sry but where  does the script go?

**Comment by Hexellate** [comment]
> Sry but where does the script go?

Looks like a python script that can be run from anywhere

**Comment by Perhelion** [comment]
@Hexellate 
Yea, I fixed this and made the script much more robust (now a own gist): https://gist.github.com/Perhelion/990b67406e0a15359ae08b97e7d415d4
@Vargtazz 
PS: If your system not get the python path (after you installed python3), search the python.exe and run from the shell (or simply put the py in the python path), eg:
`Python34\python.exe someFile.py`

**Comment by Vargtazz** [comment]
Thanks for the help. Got it working now.

**Comment by Perhelion** [comment]
I made today also a FIX-script for a annoying defect mods_registry (invalid or double registration, local instead steam location etc...)
https://gist.github.com/Perhelion/63c7d15839ec325d462add278e18f35b

**Comment by KarlSchwartz** [comment]
hello there! opened a new issue, dunno if was important, btw, mod list order it's not working since yesterday with the new launcher version
